src/main/java/cam72cam/immersiverailroading/registry/LocomotiveDieselDefinition.java

```diff
@@ -69,6 +75,12 @@ public void parseJson(JsonObject data) throws Exception {
             if (sounds.has("bell")) {
                 bell = new Identifier(ImmersiveRailroading.MODID, sounds.get("bell").getAsString()).getOrDefault(default_bell);
             }
+            if (sounds.has("motor")) {
+                motor = new Identifier(ImmersiveRailroading.MODID, sounds.get("motor").getAsString()).getOrDefault(default_motor);
+            }
+            if (sounds.has("motor2x")) {
+                motor2x = new Identifier(ImmersiveRailroading.MODID, sounds.get("motor2x").getAsString()).getOrDefault(default_motor2x);
```

Comment:
Are there scenarios where we would want additional splits between motor sounds?  Would it make sense for this to be an array of sounds instead of a "motor1": ..., "motor2": ..., "motor3": ...?

Reply:
I'm far from an expert on what different locomotive sounds should be emulated as

Reply:
it's a workaround since there's a limit on how high the pitch can go, so a sped up file is needed

Reply:
Ah, understood!  We still might want to change how this is defined in the json.

What do you think about:
```json
  "motor_progression": [
    "immersiverailroading:.../loco_1x.ogg",
    "immersiverailroading:.../loco_2x.ogg",
    "immersiverailroading:.../loco_4x.ogg"
  ]
```

Where each subsequent file is required to be 2x of the previous one?